digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala

```diff
@@ -169,7 +169,7 @@ class VVHClient(vvhRestApiEndPoint: String) {
     withLimitFilter("ROADNUMBER", roadNumbers._1, roadNumbers._2, includeAllPublicRoads)
   }
 
-  private def withRoadNumbersFilter(roadNumbers: Seq[(Int, Int)], includeAllPublicRoads: Boolean, filter: String = ""): String = {
+  private def withRoadNumbersFilter(roadNumbers: Seq[(Int, Int)], includeAllPublicRoads: Boolean = false, filter: String = ""): String = {
```

Comment:
I think it's better to not have a default value here: it's internal and whether the boolean is set to true or false has a severe impact on the functionality as a whole. If it was for compatibility (and it wasn't private def) then it'd be good to have it here.